{"content": {"suites": {"0": {"status": {"total_pass": 2, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 1, "total_fail": 1, "total_error": 0}, "tests": {"0": {"status": "FAIL", "message": "   selenium.common.exceptions.InvalidArgumentException: Message: invalid argument: invalid 'id'\n     (Session info: headless chrome=91.0.4472.77)\n", "test_name": "test_task_one", "rerun": "1"}, "1": {"status": "PASS", "message": "", "test_name": "test_task_two", "rerun": "0"}, "2": {"status": "PASS", "message": "", "test_name": "test_task_three", "rerun": "0"}}, "suite_name": "tests/web/test_congenico_coding_challenge.py"}}}, "date": "October 16, 2022", "start_time": 1665946803.044024, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "2", "fail": "1", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "1"}, "total_tests": "3"}